### PR TITLE
dm: add newly added lightning kv configurations

### PR DESCRIPTION
**Pull request by lichunzhu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
https://github.com/pingcap/tiflow/issues/8654 add configurations for newly added lightning kv configurations

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiflow/pull/8653

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by D3Hunter — APPROVED** [review]

File: dm/task-configuration-file-full.md [hunk 1]
```diff
@@ -149,6 +149,18 @@ loaders:                             # load 处理单元的运行配置参数
...
+    # - "optional"。表示导入完成后进行数据分析，如果操作失败会打印 warn 日志，任务不会暂停。
+    # - "off"。表示导入完成后不进行数据分析。
+    # Analyze 仅影响统计数据，建议在大部分场景下无需开启 Analyze。
+    analyze: "off"
+    # physical import Mode TiKV 写入 KV 数据的并发度。当 dm-worker 和 TiKV 直接网络传输速度超过万兆的时候，可以适当增加这个值。
+    # range-concurrency: 16
+    # Physical Import Mode 向 TiKV 发送 KV 时是否启用压缩。目前只支持 Gzip 压缩算法，可填写 "gzip" 或者 "gz"。默认不启用压缩。
+    # compress-kv-pairs: ""
+    # pd-server 的地址，填一个即可。为空时默认使用 TiDB 中查询到的 pd 地址信息。
+    # pd-addr: "192.168.0.1:2379"
```
Comment (Oreoxmt) [inline dm/task-configuration-file-full.md:163]:
```suggestion
    # 配置在 CHECKSUM 结束后是否对所有表执行 `ANALYZE TABLE <table>` 操作。
    # - "required"（默认值）。表示导入完成后进行 ANALYZE 操作，操作失败时任务暂停，需要用户手动处理。
    # - "optional"。表示导入完成后进行 ANALYZE 操作，操作失败时输出警告日志，任务不会暂停。
    # - "off"。表示导入完成后不进行 ANALYZE 操作。
    # ANALYZE 只影响统计数据，在大部分场景下建议不开启 ANALYZE。
    analyze: "off"
    # Physical Import Mode 向 TiKV 写入 KV 数据的并发度。当 dm-worker 和 TiKV 网络传输速度超过万兆时，可适当增加这个值。
    # range-concurrency: 16
    # Physical Import Mode 向 TiKV 发送 KV 数据时是否启用压缩。目前仅支持 Gzip 压缩算法，可填写 "gzip" 或 "gz"。默认不启用压缩。
    # compress-kv-pairs: ""
    # PD server 的地址，填一个即可。为空时默认使用 TiDB 查询到的 PD 地址信息。
    # pd-addr: "192.168.0.1:2379"
```

File: dm/task-configuration-file-full.md [hunk 2]
```diff
@@ -149,17 +149,17 @@ loaders:                             # load 处理单元的运行配置参数
...
     # Checksum 对比失败通常表示导入异常（数据丢失或数据不一致），因此建议总是开启 Checksum。
     checksum-physical: "required"
-    # 配置是否在 CHECKSUM 结束后对所有表逐个执行 `ANALYZE TABLE <table>` 操作。
-    # - "required"（默认值）。表示导入完成后进行 Analyze 操作，如果操作失败会让任务暂停，需要用户手动处理。
-    # - "optional"。表示导入完成后进行数据分析，如果操作失败会打印 warn 日志，任务不会暂停。
-    # - "off"。表示导入完成后不进行数据分析。
-    # Analyze 仅影响统计数据，建议在大部分场景下无需开启 Analyze。
+    # 配置在 CHECKSUM 结束后是否对所有表执行 `ANALYZE TABLE <table>` 操作。
+    # - "required"（默认值）。表示导入完成后进行 ANALYZE 操作，操作失败时任务暂停，需要用户手动处理。
+    # - "optional"。表示导入完成后进行 ANALYZE 操作，操作失败时输出警告日志，任务不会暂停。
```
Comment (ran-huang) [inline dm/task-configuration-file-full.md:154]:
```suggestion
    # - "required"（默认值）。表示导入完成后进行 ANALYZE 操作，ANALYZE 操作失败时任务暂停，需要用户手动处理。
    # - "optional"。表示导入完成后进行 ANALYZE 操作，ANALYZE 操作失败时输出警告日志，任务不会暂停。
```

File: dm/task-configuration-file-full.md [hunk 3]
```diff
@@ -149,17 +149,17 @@ loaders:                             # load 处理单元的运行配置参数
...
+    # ANALYZE 只影响统计数据，在大部分场景下建议不开启 ANALYZE。
     analyze: "off"
-    # physical import Mode TiKV 写入 KV 数据的并发度。当 dm-worker 和 TiKV 直接网络传输速度超过万兆的时候，可以适当增加这个值。
+    # Physical Import Mode 向 TiKV 写入 KV 数据的并发度。当 dm-worker 和 TiKV 网络传输速度超过万兆时，可适当增加这个值。
     # range-concurrency: 16
-    # Physical Import Mode 向 TiKV 发送 KV 时是否启用压缩。目前只支持 Gzip 压缩算法，可填写 "gzip" 或者 "gz"。默认不启用压缩。
+    # Physical Import Mode 向 TiKV 发送 KV 数据时是否启用压缩。目前仅支持 Gzip 压缩算法，可填写 "gzip" 或 "gz"。默认不启用压缩。
     # compress-kv-pairs: ""
-    # pd-server 的地址，填一个即可。为空时默认使用 TiDB 中查询到的 pd 地址信息。
+    # PD server 的地址，填一个即可。为空时默认使用 TiDB 查询到的 PD 地址信息。
```
Comment (ran-huang) [inline dm/task-configuration-file-full.md:162]:
```suggestion
    # PD server 的地址，填一个即可。该值为空时，默认使用 TiDB 查询到的 PD 地址信息。
```

**Review by ran-huang — COMMENTED** [review]
rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

**Pull request merged**